From 6c4a741f4c8e5680983ab8c603c672b91676c624 Mon Sep 17 00:00:00 2001
From: Bogdan Purcareata <bogdan.purcareata@nxp.com>
Date: Fri, 9 Feb 2018 15:01:29 +0200
Subject: [PATCH 548/767] staging: fsl-dpaa2/eth: Remove Kconfig dependency on
 QBMAN_DEBUG

The DPIO implementation no longer uses this Kconfig option. DPAA2 eth
debugfs entries are dependent on this, so remove it.

Signed-off-by: Bogdan Purcareata <bogdan.purcareata@nxp.com>
[Xulin: Original patch taken from NXP LSDK-18.09.]
Signed-off-by: Xulin Sun <xulin.sun@windriver.com>
---
 drivers/staging/fsl-dpaa2/Kconfig | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/staging/fsl-dpaa2/Kconfig b/drivers/staging/fsl-dpaa2/Kconfig
index d0a1f8c1dd80..2ea561f488c0 100644
--- a/drivers/staging/fsl-dpaa2/Kconfig
+++ b/drivers/staging/fsl-dpaa2/Kconfig
@@ -37,7 +37,7 @@ config FSL_DPAA2_ETH_USE_ERR_QUEUE
 
 # QBMAN_DEBUG requires some additional DPIO APIs
 config FSL_DPAA2_ETH_DEBUGFS
-	depends on DEBUG_FS && FSL_QBMAN_DEBUG
+	depends on DEBUG_FS
 	bool "Enable debugfs support"
 	default n
 	---help---
-- 
2.17.0

